src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates-endpoint.json

```diff
@@ -0,0 +1,220 @@
+                }
+              ]
+            },
+            "$._links.self.href": {
+              "matchers": [
+                {
+                  "regex": "http.*:\/\/.*\/v1\/api\/accounts\/123456\/mandates\\?page=1.*"
+                }
+              ]
+            },
+            "$._links.last_page.href": {
+              "matchers": [
+                {
+                  "regex": "http.*:\/\/.*\/v1\/api\/accounts\/123456\/mandates\\?page=1.*"
+                }
+              ]
+            },
+            "$._links.first_page.href": {
+              "matchers": [
+                {
+                  "regex": "http.*:\/\/.*\/v1\/api\/accounts\/123456\/mandates\\?page=1.*"
+                }
+              ]
+            },
+            "$.results[*].mandate_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].provider_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].reference": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].return_url": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].payment_provider": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].bank_statement_reference": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].created_date": {
+              "matchers": [
+                {
+                  "date": "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+                }
+              ]
+            },
+            "$.results[*].payer.name": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].payer.email": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].state.status": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$.results[*].state.details": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  ],
+  "metadata": {
+    "pact-specification": {
+      "version": "3.0.0"
+    },
+    "pact-jvm": {
+      "version": "3.5.16"
+    }
+  }
+}
```

Comment:
If you want to double check what DDConnector currently returns then you can play around with pay local and postman/curl, let me know if I can help...

```
{
    "total": 2,
    "count": 1,
    "page": 2,
    "results": [
        {
            "state": {
                "finished": false,
                "details": "example details",
                "status": "created"
            },
            "mandate_id": "g68qrb0ucvpfd3jtochq1rt2g5",
            "return_url": "www.google.com",
            "links": [
                {
                    "rel": "self",
                    "method": "GET",
                    "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates/g68qrb0ucvpfd3jtochq1rt2g5"
                },
                {
                    "rel": "next_url",
                    "method": "GET",
                    "href": "http://localhost:10200/secure/01dd76e0-c591-451b-bff5-2afef3d5bdfc"
                },
                {
                    "rel": "next_url_post",
                    "method": "POST",
                    "href": "http://localhost:10200/secure",
                    "type": "application/x-www-form-urlencoded",
                    "params": {
                        "chargeTokenId": "01dd76e0-c591-451b-bff5-2afef3d5bdfc"
                    }
                }
            ],
            "service_reference": "aReference",
            "created_date": "2019-07-11T12:30:48.317Z",
            "payment_provider": "sandbox"
        }
    ],
    "_links": {
        "self": {
            "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates?page=2&display_size=1"
        },
        "first_page": {
            "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates?page=1&display_size=1"
        },
        "last_page": {
            "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates?page=2&display_size=1"
        },
        "prev_page": {
            "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates?page=1&display_size=1"
        }
    }
}
```